### PR TITLE
pythonlib.v0.12.0

### PR DESCRIPTION
**Pull request by xclerc**

### CODE REVIEW

File: packages/pythonlib/pythonlib.v0.12.0/opam [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml"      {>= "4.07.0"}
+  "base"       {>= "v0.12" & < "v0.13"}
+  "ppx_jane"   {>= "v0.12" & < "v0.13"}
+  "ppx_python" {>= "v0.12" & < "v0.13"}
+  "stdio"      {>= "v0.12" & < "v0.13"}
+  "typerep"    {>= "v0.12" & < "v0.13"}
+  "dune"       {build & >= "1.5.1"}
```
Comment (kit-ty-kate) [inline packages/pythonlib/pythonlib.v0.12.0/opam]:
```suggestion
  "dune"       {>= "1.5.1"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

**Comment by kit-ty-kate** [comment]
otherwise everything seems to be good

File: packages/pythonlib/pythonlib.v0.12.0/opam (hunk 1 above)
Comment (xclerc) [reply packages/pythonlib/pythonlib.v0.12.0/opam]: OK, sorry for that.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**